src/python/zquantum/core/interfaces/backend_test.py

```diff
@@ -0,0 +1,160 @@
+import unittest
+import numpy as np
+from pyquil import Program
+from pyquil.gates import X, CNOT, H
+from pyquil.wavefunction import Wavefunction
+from openfermion import QubitOperator, IsingOperator
+
+from ..circuit import Circuit
+from ..measurement import Measurements, ExpectationValues
+from ..bitstring_distribution import BitstringDistribution
+
+class QuantumBackendTests(object):
```

Comment:
Why do we inherit after `object` and not `unittest.TestCase`?
Just asking.

Reply:
This was a mistake on my part. Addressed here:
5abb85a

Reply:
Actually, previous commit is reverted in this commit: e39c164. This is because `pytest` will attempt to run these base classes as test cases themselves instead of ignoring them.

Reply:
I see 👍